lldb/docs/resources/lldbgdbremote.md

```diff
@@ -1998,6 +1998,23 @@ threads (live system debug) / cores (JTAG) in your program have
 stopped and allows LLDB to display and control your program
 correctly.
 
+## qWatchpointSupportInfo
+
+Get the number of hardware watchpoints available on the remote target.
+
+```
+send packet: $qWatchpointSupportInfo:#55
+read packet: $num:4;#f9
+```
+
+`num` is the number of hardware breakpoints, it will be `0` if none are
+available.
+
+**Priority to Implement:** Low. If this packet is not supported, LLDB will assume
```

Comment:
Suggestion: `will assume` -> `may assume`. Although it's unlikely to change, this documentation could become out of date if that behavior does change in the future.

Reply:
Unless there was a new way of communicating that watchpoints are supported, and the number of them available in this process, the only two approaches lldb has are (1) ask via qWatchpointSupportInfo, or (2) send the watchpoint create packet and see what happens.  I think "will assume" is fine.

tbh this is one of those packets where it seems like a good idea but it's adding so little to lldb to have the information.  I guess the user interface can be "hey lldb watch this var", and get back "Too many watchpoints in use already" versus the no-idea-the-number-of-watchpoints behavior where lldb says "Unable to set this watchpoint" and the user needs to know that there are a finite (small) number of them that can be used simultaneously.